build.sh

```diff
@@ -28,19 +91,37 @@ push_images() {
     log "Pushing image: $tag_name"
     docker push "$tag_name"
     docker push "$alt_tag_name"
-
+    
+    # Check if a Python Dockerfile exists
     if [[ -n "$python_dockerfile" ]]; then
         log "Pushing image: $python_tag_name"
         docker push "$python_tag_name"
         docker push "$python_alt_tag_name"
+    
+    # If not, check if the CKAN version is greater than 2.10
+    elif [[ "$ckan_version_ref" =~ ^([2-9])\.([0-9]+) ]]; then
```

Comment:
We don't need this `elif` right? Otherwise `major` and `minor` might be undefined